comfyui_embedded_docs/docs/GrokImageEditNodeV2/fa.md

```
@@ -1,32 +1,65 @@
 # Grok Image Edit
 
-## نمای کلی
+### ورودی‌ها
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)